pkgs/by-name/en/ente-web/package.nix

```diff
@@ -0,0 +1,51 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchYarnDeps
+, nodejs
+, yarnConfigHook
+, yarnBuildHook
+
+}: stdenv.mkDerivation (finalAttrs: rec {
+
+
+  pname = "ente-web";
+  version = "photos-v0.9.5";
```

Comment:
This is a bit weird. I'd use `0.9.5`, and adjust the `rev` argument accordingly.

Reply:
I used the same version/rev for the API client, should I change both then? https://github.com/NixOS/nixpkgs/pull/325659

Reply:
> I used the same version/rev for the API client, should I change both then? #325659

Hmmm... I don't agree with what happened in #325659 . Do you know how to explain upstream's policy regarding their Git tags?

Reply:
They have multiple apps in the same monorepo, the tags are for the specific software+version if I'm not mistaken. I'm not affiliated with the developers in any way though, so I can just infer that from past public releases.

Reply:
> They have multiple apps in the same monorepo, the tags are for the specific software+version if I'm not mistaken. I'm not affiliated with the developers in any way though, so I can just infer that from past public releases.

That's what I thought as well, but what confuses me is that you used the same tag for both the `museum` package and for this package...

Reply:
In general, I'd encourage a consistent `src` approach for all packages by the `ente-io` stack.. See also this line from [`pkgs/README.md`](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md):

> - The version attribute must start with a digit e.g., "0.3.1rc2".

If possible, I'd use the same `src` for both building `museum` and `ente-web`.

Reply:
> That's what I thought as well, but what confuses me is that you used the same tag for both the museum package and for this package...

That is correct I think. The `photos` tags is for both backend (museum) and fronend (ente-web) as far as I can tell. The other tags are `auth` (different software) and `cli` for the command-line tool.

I will change the version attribute to just the digits for ente-web and submit a separate PR for museum

Reply:
Fixed now. If I should use a different tag, let me know but I think this is correct.

Reply:
> I will change the version attribute to just the digits for ente-web and submit a separate PR for museum

You can add a commit for `museum` here too - it will be easier to review it that way.

Reply:
Will do after rebasing when the hooks are merged. This branch doesn't include museum yet.

Reply:
Museum fixed in separate commit.